code/game/objects/items/weapons/nt_melee.dm

```diff
@@ -96,19 +136,112 @@
 		icon_state = initial(icon_state)
 	..()
 
+/obj/item/weapon/tool/sword/nt/scourge/apply_hit_effect(mob/living/carbon/human/target, mob/living/user, hit_zone)
+	. = ..()
+	if(ishuman(target))
+		var/mob/living/carbon/human/O = target
+		target.stun_effect_act(stun, agony, hit_zone, src)
+		O.say(pick("OH", "LORD", "MERCY", "SPARE", "ME", "HAVE", "PLEASE"))
+
+/obj/item/weapon/tool/sword/nt/spear
+	name = "NT Pilum"
+	desc = "A saint looking short spear, designed for use with a shield or as a throwing weapon. The spear-tip usually breaks after being thrown at a target, but it can be welded into shape again."
+	icon_state = "nt_spear"
+	item_state = "nt_spear"
+	wielded_icon = "nt_spear_wielded"
+	force = 24
+	force_wielded_multiplier = 1.08
+	var/tipbroken = FALSE
+	w_class = ITEM_SIZE_HUGE
+	slot_flags = SLOT_BACK | SLOT_BELT 
+	throwforce = 75 
+	armor_penetration = ARMOR_PEN_HALF
+	throw_speed = 3
+	price_tag = 150
+	matter = list(MATERIAL_BIOMATTER = 10, MATERIAL_STEEL = 5) // easy to mass-produce and arm the faithful
+
+/obj/item/weapon/tool/sword/nt/spear/equipped(mob/living/W)
+	. = ..()
+	if(is_held() && is_neotheology_disciple(W))
+		embed_mult = 0.1
+	else
+		embed_mult = initial(embed_mult)
+	if(isBroken)
+		force = WEAPON_FORCE_NORMAL
+		throwforce = WEAPON_FORCE_HARMLESS
+
+/obj/item/weapon/tool/sword/nt/spear/dropped(mob/living/W)
+	embed_mult = 300
+	..()
+
+/obj/item/weapon/tool/sword/nt/spear/on_embed(mob/user)
+	. = ..()
+	tipbroken = TRUE
+
+/obj/item/weapon/tool/sword/nt/spear/examine(mob/user)
+	. = ..()
+	if (tipbroken)
+		to_chat(user, SPAN_WARNING("\The [src] is broken. It looks like it could be repaired with a welder."))
+
+/obj/item/weapon/tool/sword/nt/spear/attackby(obj/item/I, var/mob/user)
+	. = ..()
+	if (I.has_quality(QUALITY_WELDING))
+		if(I.use_tool(user, src, WORKTIME_FAST, QUALITY_WELDING, FAILCHANCE_EASY, STAT_MEC))
+			to_chat(user, SPAN_NOTICE("You repair \the damaged spear-tip."))
+			tipbroken = FALSE
+
+
 /obj/item/weapon/shield/riot/nt
-	name = "NT Shield"
-	desc = "A saint looking shield, let the God protect you."
+	name = "NT Scutum"
+	desc = "A saint looking shield, let the God protect you. Has several leather straps on the back to hold melee weapons."
 	icon = 'icons/obj/nt_melee.dmi'
 	icon_state = "nt_shield"
 	item_state = "nt_shield"
-	force = WEAPON_FORCE_DANGEROUS
-	armor = list(melee = 20, bullet = 20, energy = 20, bomb = 0, bio = 0, rad = 0)
 	matter = list(MATERIAL_BIOMATTER = 25, MATERIAL_STEEL = 5, MATERIAL_PLASTEEL = 2)
 	aspects = list(SANCTIFIED)
 	spawn_blacklisted = TRUE
 	price_tag = 1000
 	base_block_chance = 40
+	item_flags = DRAG_AND_DROP_UNEQUIP
+	var/obj/item/weapon/storage/internal/container
+	var/storage_slots = 3
```

Comment:
1 slot at most ,  NT having extra-embeddy spears and being able to store 3 of them inside a shield doesn't seem that good to balance

Reply:
Carrying a shield is already a heavy commitment since you either need to hold it in your hand all the time or sacrifice your backpack, so I think some more storage space is appropriate here.

Reply:
Carrying a shield reduces all damage you take , its not that heavy of a commitment since you can just hold it.